stock_picking_invoice_product_group/models/account_invoice.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Alex Comba - Agile Business Group
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from openerp import models, fields
+
+
+class AccountInvoice(models.Model):
+    _inherit = 'account.invoice'
+
+    related_picking_ids = fields.Many2many(
```

Comment:
@pedrobaeza ```stock_picking_invoice_link``` only depends on ```stock_account``` and in the standard case the relation between invoices and pickings can only be one2many: 1 picking can only refer to  1 invoice.

Modules that change this relation, like this one or ```account_invoice_merge``` should handle themselves the relation, if needed. Or "bridge" module should be created (one could be named  ```account_invoice_merge_picking_link```, for instance)

I would not change ```stock_picking_invoice_link``` structure, having to change DB data and modules that depend on it, to support modules which could be not used with it